conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
> > Actually I'm telling runC to run one container with a given bundle path and runC explicitly ignores the second argument.
> 
> I don't know where you got that impression from.

I was assuming that runC [complies with the OCI runtime spec](https://github.com/opencontainers/runc#releases) and that `runc create <arg1> <arg2>` would be interpreted as defined here:

https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#create

but after reading the runC code, I realize these are incorrect assumptions.

@mrunalp proposes to change the spec, but I would personally prefer to see runC being fixed. Both solutions have pros and cons, so it's really up to the runC and OCI specs maintainers to decide what's best. I guess the OCI spec still being under rc gives some flexibility...